### ISSUE TITLE
Error on Client.prepare_authorization_request

### ISSUE THREAD

**Issue by pyworkshop** [oauthlib/oauthlib#301]
CODE:
auth_url = self.prepare_request_uri(  
        authorization_url, redirect_uri=**self.redirect_uri**,
        scope=self.scope, state=self.state, **kwargs)     

SHOULD BE **self.redirect_url**

**Comment by JonnyFunFun** [comment]
I encountered the same issue, as well as in `prepare_refresh_token_request`:

Around line 300:
```
body = self._client.prepare_refresh_body(body=body,
                refresh_token=refresh_token, scope=self.scope, **kwargs)
```

However there is no `_client` attribute.  I submitted a pull request with the change in #324 but it appears that all tests fail after this, so I'm not sure what to do with it.

**Comment by JonathanHuot** [comment]
Fixed since #324 was merged
